src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -406,61 +491,12 @@ HashSet<ITypeSymbol> typeStack
 
 			return DoInspectType(
 				field.Type,
+				hostSymbol,
 				MutabilityInspectionFlags.Default,
 				typeStack
 			).WithPrefixedMember( field.Name );
 		}
 
-		/// <summary>

```

Comment:
Just moved to the bottom of the file.

Reply:
Looks like you changed the method too.  Did you mean to?

![image](https://user-images.githubusercontent.com/12101120/42286495-1a7a9c02-7f81-11e8-940d-7e894d12e9a4.png)

Reply:
Actually looks like a merge conflict.   Do smaller PRs.  Move methods on their own.  Not as a part of other PRs